public/app/core/services/backend_srv.ts

```diff
@@ -94,10 +93,6 @@ export class BackendSrv implements BackendService {
   }
 
   private async initGrafanaDeviceID() {
-    if (config.buildInfo?.edition === GrafanaEdition.OpenSource) {
-      return;
-    }
-
```

Comment:
I wonder why this header has not been added previously, i.e. is there a reason for not adding it?

Reply:
the decision was made to do it in enterprise. the decision was made to do this for opensource now. no particular reason than that

Reply:
There were questions if this would be useful for OSS and if it made sense to display. The final decision was yes, having an overview of your anon users in OSS is useful